src/app/code/community/Alma/Installments/Model/System/Config/Backend/PnxConfig.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@magento.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Magento to newer
+ * versions in the future. If you wish to customize Magento for your
+ * needs please refer to http://www.magento.com for more information.
+ *
+ * @category    Mage
+ * @package     Mage_CatalogInventory
+ * @copyright  Copyright (c) 2006-2020 Magento, Inc. (http://www.magento.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Adminhtml catalog inventory "Minimum Qty Allowed in Shopping Cart" field
+ *
+ * @category   Mage
+ * @package    Mage_CatalogInventory
+ * @author     Magento Core Team <core@magentocommerce.com>
+ */
+class Alma_Installments_Model_System_Config_Backend_PnxConfig extends Mage_Adminhtml_Model_System_Config_Backend_Serialized_Array
+{
+    /**
+     * @var Mage_Core_Helper_Abstract|null
+     */
+    private $feePlansHelper;
+    public function __construct()
+    {
+        $this->feePlansHelper = Mage::helper('alma/FeePlansHelper');
+
+        parent::__construct();
+    }
+
+
+    public function _afterLoad()
```

Comment:
can you add phpdoc with a description of what is done inside this inherited method please

you can also add `@override` if this method is not abstract in parent class

then, I think this method should be `protected` .... WDYT ?